source/js/components/petition/petition.scss

```diff
@@ -17,7 +17,7 @@
     background: $white;
     padding: 15px;
     width: 100%;
-    box-shadow: 0 -1px 3px rgba(0, 0, 0, 0.15);
+    box-shadow: 0 -1px 3px transparentize($black, 0.85);
```

Comment:
Is this a real function, or did we come up with it? If the latter, can we rename it to something like `transparent()` instead?

Reply:
it's SASS/SCSS' built-in function: https://sass-lang.com/documentation/functions/color#transparentize

note: its opposite function is called [opacify](https://sass-lang.com/documentation/functions/color#opacify)